### PR TITLE
Format YAML files for ansible-lint

### LINKED ISSUE #1313: YAML formatting should be cleaned up to linter standards
**What steps did you take and what happened:**

Run `make -C images/capi lint` and see that there are numerous warnings from `ansible-lint` that the project is ignoring.

In particular, there are many formatting warnings. Many of these boil down to what the linter considers basic YAML linting (removing unnecessary quotes, ensuring document separators `---`, and so on). It would be helpful to fix just those separately.

**What did you expect to happen:**

Running `make -C images/capi lint` shouldn't result in any basic formatting warnings.

**Anything else you would like to add:**

One way to accomplish this:

```shell
ansible-lint --fix=none images/capi/ansible
make -C images/capi lint-ignore
git add images/capi
git commit -m "Format YAML files for ansible-lint"
```

These commands:
- Tell the linter not to fix a specific type of warning, so it just does its basic formatting
- Regenerate the `.ansible-lint-ignore` file so these warnings won't be ignored in the future
- Add all the changed files to version control and commit them.

Then run `make -C images/capi lint` to ensure it's passing, and create a PR.

**Environment:**

N/A

/kind bug
/help
/good-first-issue

### PR DESCRIPTION
**Pull request by FaithKovi**
What this PR does / why we need it: fixes #1313 

Which issue(s) this PR fixes (optional, in fixes #<issue number>(, fixes #<issue_number>, ...) format, will close the issue(s) when PR gets merged): Fixes #

**Additional context**
Add any other context for the reviewers
YAML formatting should be cleaned up to linter standards
Cleans up YAML formatting to linter standards

### CODE REVIEW

**Review by drew-viles — COMMENTED** [review]
Hi! Thanks for contributing this.

I've had a quick look through and it looks good initially. However, I have noticed that there are a number of potentially accidental line breaks in some of the changes. I'll mark all of these with a comment `New line break`.

If you can resolve those and I'll do some testing on my side in the morning (UK time) and let you know how I get on with it.

File: images/capi/ansible/roles/firstboot/meta/main.yml [hunk 1]
```diff
@@ -21,6 +21,7 @@ dependencies:
 
   - role: setup
     vars:
-      rpms: "{{ ( ( common_rpms + rh7_rpms + lookup('vars', 'common_' + build_target + '_rpms') ) if (ansible_os_family == 'RedHat' and ansible_distribution_major_version == '7') else ( common_rpms + rh8_rpms + lookup('vars', 'common_' + build_target + '_rpms') ) ) }}"
+      rpms: "{{ ( ( common_rpms + rh7_rpms + lookup('vars', 'common_' + build_target + '_rpms') ) if (ansible_os_family == 'RedHat' and ansible_distribution_major_version
+        == '7') else ( common_rpms + rh8_rpms + lookup('vars', 'common_' + build_target + '_rpms') ) ) }}"
```
Comment (drew-viles) [inline images/capi/ansible/roles/firstboot/meta/main.yml:25]: New line break

File: images/capi/ansible/roles/kubernetes/tasks/photon.yml [hunk 2]
```diff
@@ -21,4 +21,5 @@
 - name: Install Kubernetes
   command: tdnf install {{ packages }} --nogpgcheck -y
   vars:
-    packages: "kubelet-{{ kubernetes_rpm_version }} kubeadm-{{ kubernetes_rpm_version }} kubectl-{{ kubernetes_rpm_version }} kubernetes-cni-{{ kubernetes_cni_rpm_version }}"
+    packages: kubelet-{{ kubernetes_rpm_version }} kubeadm-{{ kubernetes_rpm_version }} kubectl-{{ kubernetes_rpm_version }} kubernetes-cni-{{ kubernetes_cni_rpm_version
+      }}
```
Comment (drew-viles) [inline images/capi/ansible/roles/kubernetes/tasks/photon.yml:25]: New line break

File: images/capi/ansible/roles/node/defaults/main.yml [hunk 3]
```diff
@@ -108,22 +108,21 @@ common_virt_photon_rpms:
...
   - linux-cloud-tools-generic
   - linux-tools-generic
 
 common_raw_photon_rpms: []
-
 # photon and flatcar do not have backward compatibility for legacy distro behavior for sysctl.conf by default
 # as it uses systemd-sysctl. set this var so we can use for sysctl conf file value.
-sysctl_conf_file: "{{ '/etc/sysctl.d/99-sysctl.conf' if ansible_os_family in ['Common Base Linux Mariner', 'Flatcar', 'VMware Photon OS'] else '/etc/sysctl.conf' }}"
+sysctl_conf_file: "{{ '/etc/sysctl.d/99-sysctl.conf' if ansible_os_family in ['Common Base Linux Mariner', 'Flatcar', 'VMware Photon OS'] else '/etc/sysctl.conf'
+  }}"
```
Comment (drew-viles) [inline images/capi/ansible/roles/node/defaults/main.yml:119]: New line break

File: images/capi/ansible/roles/node/meta/main.yml [hunk 4]
```diff
@@ -27,15 +27,17 @@ dependencies:
 
   - role: setup
     vars:
-      rpms: "{{ (common_photon_rpms + lookup('vars', 'photon_' + ansible_distribution_major_version + '_rpms' ) + lookup('vars', 'common_' + build_target + '_photon_rpms')) }}"
+      rpms: "{{ (common_photon_rpms + lookup('vars', 'photon_' + ansible_distribution_major_version + '_rpms' ) + lookup('vars', 'common_' + build_target + '_photon_rpms'))
+        }}"
```
Comment (drew-viles) [inline images/capi/ansible/roles/node/meta/main.yml:31]: New line break

File: images/capi/ansible/roles/node/meta/main.yml [hunk 5]
```diff
@@ -27,15 +27,17 @@ dependencies:
...
     when: ansible_distribution == "VMware Photon OS"
 
   - role: setup
     vars:
-      rpms: "{{ ( ( common_rpms + rh7_rpms + lookup('vars', 'common_' + build_target + '_rpms') ) if (ansible_os_family == 'RedHat' and ansible_distribution_major_version == '7') else ( common_rpms + rh8_rpms + lookup('vars', 'common_' + build_target + '_rpms') ) ) }}"
+      rpms: "{{ ( ( common_rpms + rh7_rpms + lookup('vars', 'common_' + build_target + '_rpms') ) if (ansible_os_family == 'RedHat' and ansible_distribution_major_version
+        == '7') else ( common_rpms + rh8_rpms + lookup('vars', 'common_' + build_target + '_rpms') ) ) }}"
```
Comment (drew-viles) [inline images/capi/ansible/roles/node/meta/main.yml:37]: New line break

File: images/capi/ansible/roles/node/meta/main.yml [hunk 6]
```diff
@@ -27,15 +27,17 @@ dependencies:
...
       debs: "{{ common_debs +  lookup('vars', 'common_' + build_target + '_debs') }}"
-    when: ansible_distribution != "VMware Photon OS" and ansible_distribution != "Amazon" and not (packer_builder_type == "oracle-oci" and ansible_architecture == "aarch64") and
-      not packer_builder_type is search('qemu')
+    when: ansible_distribution != "VMware Photon OS" and ansible_distribution != "Amazon" and not (packer_builder_type == "oracle-oci" and ansible_architecture ==
+      "aarch64") and not packer_builder_type is search('qemu')
```
Comment (drew-viles) [inline images/capi/ansible/roles/node/meta/main.yml:40]: New line break

File: images/capi/ansible/roles/providers/tasks/main.yml [hunk 7]
```diff
@@ -132,8 +127,5 @@
     state: started
     daemon_reload: true
     name: chronyd
-  when: (packer_builder_type.startswith('amazon') or
-    packer_builder_type.startswith('azure') or
-    packer_builder_type is search('vmware') or
-    packer_builder_type is search('vsphere')) and
-    ansible_os_family != "Flatcar"
+  when: (packer_builder_type.startswith('amazon') or packer_builder_type.startswith('azure') or packer_builder_type is search('vmware') or packer_builder_type is
+    search('vsphere')) and ansible_os_family != "Flatcar"
```
Comment (drew-viles) [inline images/capi/ansible/roles/providers/tasks/main.yml:131]: New line break

File: images/capi/ansible/roles/python/tasks/flatcar.yml [hunk 8]
```diff
@@ -21,10 +21,11 @@
     - pypy_installed.stdout_lines[0] == "false"
   block:
     - name: Download pypy archive
-      raw: "curl https://github.com/squeaky-pl/portable-pypy/releases/download/pypy{{ pypy_python_version }}-{{ pypy_version }}/pypy{{ pypy_python_version }}-{{ pypy_version }}-linux_x86_64-portable.tar.bz2 -L --output {{ pypy_download_path }}"
+      raw: curl https://github.com/squeaky-pl/portable-pypy/releases/download/pypy{{ pypy_python_version }}-{{ pypy_version }}/pypy{{ pypy_python_version }}-{{ pypy_version
+        }}-linux_x86_64-portable.tar.bz2 -L --output {{ pypy_download_path }}
```
Comment (drew-viles) [inline images/capi/ansible/roles/python/tasks/flatcar.yml:25]: New line break

File: images/capi/ansible/windows/roles/kubernetes/tasks/sc.yml [hunk 9]
```diff
@@ -17,11 +17,15 @@
...
 
 - name: Install kubelet as service
   win_service:
     name: kubelet
     start_mode: manual
-    path: '{{ kubernetes_install_path }}\kubelet.exe --windows-service --cert-dir={{ systemdrive.stdout | trim }}/var/lib/kubelet/pki --config={{ systemdrive.stdout | trim }}/var/lib/kubelet/config.yaml --bootstrap-kubeconfig={{ systemdrive.stdout | trim }}/etc/kubernetes/bootstrap-kubelet.conf --kubeconfig={{ systemdrive.stdout | trim }}/etc/kubernetes/kubelet.conf --hostname-override=$(hostname) --pod-infra-container-image=`"{{ pause_image }}`" --enable-debugging-handlers --cgroups-per-qos=false --enforce-node-allocatable=`"`" --resolv-conf=`"`" --log-dir={{ systemdrive.stdout | trim }}/var/log/kubelet --logtostderr=false {{ additional_kubelet_args if additional_kubelet_args is defined }}'
+    path: '{{ kubernetes_install_path }}\kubelet.exe --windows-service --cert-dir={{ systemdrive.stdout | trim }}/var/lib/kubelet/pki --config={{ systemdrive.stdout
+      | trim }}/var/lib/kubelet/config.yaml --bootstrap-kubeconfig={{ systemdrive.stdout | trim }}/etc/kubernetes/bootstrap-kubelet.conf --kubeconfig={{ systemdrive.stdout
```
Comment (drew-viles) [inline images/capi/ansible/windows/roles/kubernetes/tasks/sc.yml:28]: New line break

File: images/capi/ansible/windows/roles/kubernetes/tasks/sc.yml [hunk 10]
```diff
@@ -17,11 +17,15 @@
...
+      | trim }}/etc/kubernetes/kubelet.conf --hostname-override=$(hostname) --pod-infra-container-image=`"{{ pause_image }}`" --enable-debugging-handlers --cgroups-per-qos=false
```
Comment (drew-viles) [inline images/capi/ansible/windows/roles/kubernetes/tasks/sc.yml:29]: New line break

File: images/capi/ansible/windows/roles/kubernetes/tasks/sc.yml [hunk 11]
```diff
@@ -17,11 +17,15 @@
...
+      --enforce-node-allocatable=`"`" --resolv-conf=`"`" --log-dir={{ systemdrive.stdout | trim }}/var/log/kubelet --logtostderr=false {{ additional_kubelet_args
```
Comment (drew-viles) [inline images/capi/ansible/windows/roles/kubernetes/tasks/sc.yml:30]: New line break

File: images/capi/ansible/windows/roles/kubernetes/tasks/sc.yml [hunk 12]
```diff
@@ -17,11 +17,15 @@
...
+      if additional_kubelet_args is defined }}'
```
Comment (drew-viles) [inline images/capi/ansible/windows/roles/kubernetes/tasks/sc.yml:31]: New line break

File: images/capi/ansible/windows/roles/systemprep/defaults/main.yml [hunk 13]
```diff
@@ -13,5 +13,6 @@
 # limitations under the License.
 ---
 windows_updates_kbs_numbers: "{{ windows_updates_kbs.split() if (windows_updates_kbs is defined) and (windows_updates_kbs | length > 0) else [] }}"
-windows_updates_category_names: "{{ windows_updates_categories.split() if (windows_updates_categories is defined) and (windows_updates_categories | length > 0) else [] }}"
+windows_updates_category_names: "{{ windows_updates_categories.split() if (windows_updates_categories is defined) and (windows_updates_categories | length > 0) else
+  [] }}"
```
Comment (drew-viles) [inline images/capi/ansible/windows/roles/systemprep/defaults/main.yml:17]: New line break

**Review by drew-viles — COMMENTED** [review]
There we go - I don't think anything is Earth shattering really, it's more just keeping with format that exists in those locations already. 

Thanks again!

**Comment by drew-viles** [comment]
No problem at all - I realised after I'd put the review in that it was probably the linter limiting line lengths or something 😄 

Cool, let me check it my side in the morning then and I'll update you then!

**Comment by AverageMarcus** [comment]
I'm actually wondering if we should add a global ignore for the `yaml[line-length]` rule? 🤔  I personally prefer the longer lines with more logical line breaks rather than the arbitrary breaks added by the linter. 

What do y'all think? (cc @mboersma )

**Comment by drew-viles** [comment]
I am inclined to agree with that. It wouldn't be so bad if it did nice formatting but it just breaks the line at a point where 80 chars is hit rather than at any point that is easy to read.

**Comment by drew-viles** [comment]
Either that or we come up with a styling guide for multiline strings ;-)

**Comment by AverageMarcus** [comment]
> Either that or we come up with a styling guide for multiline strings ;-)

Do you not have enough work to do already? 😆 

@FaithKovi Would you like to try adding the config to ignore the line-length rule into this PR or would you prefer if we handled it in a different PR? I'm happy to help out either way. 😄

**Comment by FaithKovi** [comment]
> > Either that or we come up with a styling guide for multiline strings ;-)
> 
> Do you not have enough work to do already? 😆
> 
> @FaithKovi Would you like to try adding the config to ignore the line-length rule into this PR or would you prefer if we handled it in a different PR? I'm happy to help out either way. 😄

Yes, I would like to try that. How do I go about it?

**Comment by AverageMarcus** [comment]
It might be best to first undo the changes you've already done with the line breaks.

Then you'll want to add a new file called `.ansible-lint` into the `./images/capi/` directory with the following contents:

```
skip_list:
  - yaml[line-length]
```

With that done I _think_ you should be able to re-run the fix command to have all except the line length ones fixed. (I'm not sure if it'll actually still try and fix them or not, we'll have to try and find out 😄)

At the very least, they should no longer be shown as warning or errors with that configuration in place.

**Comment by FaithKovi** [comment]
> It might be best to first undo the changes you've already done with the line breaks.
> 
> Then you'll want to add a new file called `.ansible-lint` into the `./images/capi/` directory with the following contents:
> 
> ```
> skip_list:
>   - yaml[line-length]
> ```
> 
> With that done I _think_ you should be able to re-run the fix command to have all except the line length ones fixed. (I'm not sure if it'll actually still try and fix them or not, we'll have to try and find out 😄)
> 
> At the very least, they should no longer be shown as warning or errors with that configuration in place.

okay will do that. Thanks @AverageMarcus

**Comment by mboersma** [comment]
As far as I can tell, the configuration suggested above doesn't quite work. I created an `./images/capi/.ansible-lint` file with these contents:

```yaml
skip_list:
  - yaml[line-length]
```

Then I ran the fixup command and referenced that configuration explicitly:

```shell
ansible-lint --fix=none -c images/capi/ansible-lint images/capi/ansible
```

It didn't change any files after this. I think `skip_list` is only handling tags and `yaml` matches all formatting warnings. I read the ansible-lint docs and tried several variations but couldn't get this working.

@FaithKovi I think if you manually removed the unnecessary new lines that @drew-viles pointed out above and then ran `make -C images/capi lint-fix`, it would do the right thing. I think... 😄 

Sorry this wasn't quite as simple as it seemed! Thanks for your patience.

**Comment by FaithKovi** [comment]
> As far as I can tell, the configuration suggested above doesn't quite work. I created an `./images/capi/.ansible-lint` file with these contents:
> 
> ```yaml
> skip_list:
>   - yaml[line-length]
> ```
> 
> Then I ran the fixup command and referenced that configuration explicitly:
> 
> ```shell
> ansible-lint --fix=none -c images/capi/ansible-lint images/capi/ansible
> ```
> 
> It didn't change any files after this. I think `skip_list` is only handling tags and `yaml` matches all formatting warnings. I read the ansible-lint docs and tried several variations but couldn't get this working.
> 
> @FaithKovi I think if you manually removed the unnecessary new lines that @drew-viles pointed out above and then ran `make -C images/capi lint-fix`, it would do the right thing. I think... 😄 
> 
> Sorry this wasn't quite as simple as it seemed! Thanks for your patience.

Okay I will work on this today

File: images/capi/ansible/roles/providers/tasks/main.yml (hunk 7 above)
Comment (mboersma) [reply images/capi/ansible/roles/providers/tasks/main.yml:131]: This should be fine according to ansible docs. A `when` clause can handle whitespace. I just don't understand why it reformatted it from the multi-line version that was there.

File: images/capi/ansible/windows/example.vars.yml [hunk 14]
```diff
@@ -19,20 +19,20 @@ nssm_url: https://azurek8scishared.blob.core.windows.net/nssm/nssm.exe
 gmsa_keyvault_url: https://kubernetesartifacts.azureedge.net/ccgakvplugin/v1.1.4/binaries/windows-gmsa-ccgakvplugin-v1.1.4.zip
 
 runtime: docker-ee
-docker_ee_version: "19.03.12"
-kubernetes_install_path: 'c:\k'
-windows_service_manager: 'nssm'
-pause_image: "registry.k8s.io/pause:3.9"
+docker_ee_version: 19.03.12
+kubernetes_install_path: c:\k
```
Comment (mboersma) [inline images/capi/ansible/windows/example.vars.yml:23]: Do we think this is safe? I'm not sure how backslashes and escaping work in YAML.

File: images/capi/ansible/windows/roles/kubernetes/tasks/kubelet.yml [hunk 15]
```diff
@@ -13,30 +13,30 @@
...
-  win_file:
+  ansible.windows.win_file:
     path: "{{ item }}"
     state: directory
   loop:
-    - '{{ systemdrive.stdout | trim }}\var\log\kubelet'
-    - '{{ systemdrive.stdout | trim }}\etc\kubernetes'
-    - '{{ systemdrive.stdout | trim }}\etc\kubernetes\manifests'
-    - '{{ systemdrive.stdout | trim }}\etc\kubernetes\pki'
+    - "{{ systemdrive.stdout | trim }}\\var\\log\\kubelet"
```
Comment (mboersma) [inline images/capi/ansible/windows/roles/kubernetes/tasks/kubelet.yml:20]: Based on this, maybe we should change the string I pointed to above to `"c:\\k"`.

File: images/capi/ansible/windows/roles/kubernetes/tasks/sc.yml (hunk 9 above)
Comment (mboersma) [reply images/capi/ansible/windows/roles/kubernetes/tasks/sc.yml:28]: One way to handle this is with the YAML multiline operator, so each line gets joined back together with a space:

```yaml
path: >
  {{ kubernetes_install_path }}\kubelet.exe --windows-service 
    --cert-dir={{ systemdrive.stdout | trim }}/var/lib/kubelet/pki 
    --config={{ systemdrive.stdout | trim }}/var/lib/kubelet/config.yaml 
    --bootstrap-kubeconfig={{ systemdrive.stdout | trim }}/etc/kubernetes/bootstrap-kubelet.conf 
    --kubeconfig={{ systemdrive.stdout | trim }}/etc/kubernetes/kubelet.conf 
    --hostname-override=$(hostname) --pod-infra-container-image=`"{{ pause_image }}`" 
    --enable-debugging-handlers --cgroups-per-qos=false --enforce-node-allocatable=`"`"
    --resolv-conf=`"`" --log-dir={{ systemdrive.stdout | trim }}/var/log/kubelet
    --logtostderr=false {{ additional_kubelet_args if additional_kubelet_args is defined }}
```

But frankly there's a lot of weird escaping going on here. It might be better to revert this line to the original, then run `make -C images/capi lint-ignore` to leave it as-is.

File: images/capi/ansible/windows/roles/systemprep/tasks/main.yml [hunk 16]
```diff
@@ -50,14 +50,9 @@
 # causing issues with communication over winrm and setting winrm to delayed start
 # gives Hyper-V enough time to finish configuration before having packer continue.
 - name: Set WinRm Service to delayed start
-  win_command: sc.exe config winrm start=delayed-auto
-
-# Best effort to update defender signatures
```
Comment (mboersma) [inline images/capi/ansible/windows/roles/systemprep/tasks/main.yml:55]: It removed all these comments? That seems out of scope for the linter. We should restore them, because someone took the time to write them and they effectively constitute Windows build documentation.

File: images/capi/ansible/windows/roles/systemprep/tasks/main.yml [hunk 17]
```diff
@@ -92,50 +87,48 @@
...
 # Apply HNS flags for fixes that need to be enabled via Registry
 # these eventually get turned on automatically and can be removed in future releases
 - name: Apply HNS control Flags 0x40 and 0x10 in 2022-11B patches
-  win_regedit:
+  ansible.windows.win_regedit:
     path: HKLM:\SYSTEM\CurrentControlSet\Services\hns\State
     state: present
     name: HNSControlFlag
-    data: 0x50
+    data: "0120"
```
Comment (mboersma) [inline images/capi/ansible/windows/roles/systemprep/tasks/main.yml:126]: Will this end up putting the same value in the registry? Seems dangerous to change this.

File: images/capi/packer/ova/packer-common.json [hunk 18]
```diff
@@ -23,8 +23,6 @@
   "remote_username": "",
   "skip_compaction": "false",
   "ssh_password": "builder",
-  "ssh_proxy_host": "",
```
Comment (mboersma) [inline images/capi/packer/ova/packer-common.json:26]: These lines were added in recent PR #1314 and shouldn't be removed by this change. Could you rebase the entire PR from the current `main` branch?

**Review by mboersma — COMMENTED** [review]
Overall, I think we've learned to take baby steps when making sweeping formatting changes, and that `ansible-lint --fix` doesn't behave quite as we expected.

Thanks for sticking with this @FaithKovi. I made some comments but overall this looks like good cleanup. Please let us know if we can help with anything, or if you've had enough. 😄

**Comment by mboersma** [comment]
Also, I thought the "fqcn" changes should be in a separate PR. That is, this PR shouldn't be changing `command:` to `ansible.builtin.command:`. Is that what `ansible-lint --fix=none` did?

I'm going to make a PR to update ansible-lint itself; I see there have been a few fixes related to `--fix` recently.

**Comment by FaithKovi** [comment]
> Also, I thought the "fqcn" changes should be in a separate PR. That is, this PR shouldn't be changing `command:` to `ansible.builtin.command:`. Is that what `ansible-lint --fix=none` did?
> 
> I'm going to make a PR to update ansible-lint itself; I see there have been a few fixes related to `--fix` recently.

Yes that what `ansible-lint --fix=none` did
@mboersma

**Comment by mboersma** [comment]
> that's what `ansible-lint --fix=none` did

I see. Ok, I tried this again from scratch and I think this works better:

```
$ git co main
$ git pull
$ cd images/capi
$ ansible-lint fix=none ansible
```

If I do that, I get a large set of changes, but it doesn't include any of the "fqcn" changes (maybe because it finds them in the `.ansible-lint-ignore` file in that current directory).

@FaithKovi would you be willing to start this PR over that way? We would still have the issue of "line too long" and maybe some mistaken escape characters to fix up, but it should be much easier to review.

My apologies for all this work! We made some assumptions about how `ansible-lint` worked that turned out to be wrong, so this wasn't as simple a change as we thought. But I think we can keep chipping away at it and get to a better ansible codebase.

**Comment by FaithKovi** [comment]
> > that's what `ansible-lint --fix=none` did
> 
> I see. Ok, I tried this again from scratch and I think this works better:
> 
> ```
> $ git co main
> $ git pull
> $ cd images/capi
> $ ansible-lint fix=none ansible
> ```
> 
> If I do that, I get a large set of changes, but it doesn't include any of the "fqcn" changes (maybe because it finds them in the `.ansible-lint-ignore` file in that current directory).
> 
> @FaithKovi would you be willing to start this PR over that way? We would still have the issue of "line too long" and maybe some mistaken escape characters to fix up, but it should be much easier to review.
> 
> My apologies for all this work! We made some assumptions about how `ansible-lint` worked that turned out to be wrong, so this wasn't as simple a change as we thought. But I think we can keep chipping away at it and get to a better ansible codebase.

okay @mboersma I will follow your instructions and start all over. Do I create a new PR for it?

**Comment by mboersma** [comment]
> Do I create a new PR for it?

Or you could force-push the new commit over this one. Either way is fine; whatever is easier for you.

**Comment by FaithKovi** [comment]
@mboersma 
I created a new PR for it : #1328

**Comment by mboersma** [comment]
/close

See #1328 instead.
